test/test_formatter.rb

```diff
@@ -13,12 +13,16 @@ def record
     {'message' => 'awesome', 'greeting' => 'hello'}
   end
 
+  def config_element_in_params(params)
+    config_element('test', '', params)
+  end
+
   class BaseFormatterTest < ::Test::Unit::TestCase
     include FormatterTest
 
     def test_call
       formatter = Formatter.new
-      formatter.configure({})
+      formatter.configure(config_element())
```

Comment:
I think we should use `Fluent::Test::FormatterTestDriver` along with other formatter tests.
Then we can pass `Hash` as before.

Reply:
I think it doesn't use `Fluent::Test::FormatterTestDriver` intentionally.
Because `test_call` in `BaseFormatterTestWithTestDriver` does the test you intend.

Reply:
Oh, sorry. You are right. This code should be as is. Thanks!